sample-application/sample-application/app/src/main/java/com/example/samplecredentialwallet/ui/credential/CredentialListScreen.kt

```
@@ -0,0 +1,404 @@
+        if (credentialSubject != null) {
+            credentialSubject.keys().forEach { key ->
+                val value = credentialSubject.opt(key)
+                when (value) {
+                    is String -> {
+                        if (key == "face" && value.startsWith("data:image/")) {
+                            result["faceImage"] = value.substringAfter(",")
+                        } else if (value.trim().startsWith("{") && value.contains("\"language\"") && value.contains("\"value\"")) {
+                            try {
+                                val langObj = JSONObject(value)
+                                addField(key, langObj.optString("value", ""))
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)